### PR TITLE
Simplify wix files for SwiftFormat.

### PR DESCRIPTION
**Pull request by hjyamauchi**
Use autogenerated GUIDs.
Merge the two wxs files into one.

### CODE REVIEW

File: installer-scripts/SwiftFormat.wxs [hunk 1]
```diff
@@ -2,42 +2,41 @@
   <Package
       Language="1033"
       Manufacturer="nicklockwood"
-      Name="SwiftFormat for Windows x86_64"
+      Name="SwiftFormat for Windows $(var.ProductArchitecture)"
```
Comment (tristanlabelle) [inline installer-scripts/SwiftFormat.wxs:5]: I don't think it matters but this is probably going to expand to `amd64` now. @compnerd do we care?

**Review by tristanlabelle — APPROVED** [review]
Congratulations! 🎉 You just became the new WiX expert on the team! Now @compnerd is going to assume you can fix any and all WiX issues 😉.

**Comment by tristanlabelle** [comment]
Curious, is there anything that prevents someone on amd64 from installing the arm64 SwiftFormat msi?

File: installer-scripts/SwiftFormat.wxs (hunk 1 above)
Comment (compnerd) [reply installer-scripts/SwiftFormat.wxs:5]: I think that we can drop the architecture name entirely.  It adds no value and is something I'm considering for the toolchain itself: https://github.com/apple/swift-installer-scripts/pull/196

**Comment by compnerd** [comment]
@tristanlabelle I _think_ that WI does flag that as incompatible.  If not, we should aid it in that.

File: installer-scripts/SwiftFormat.wxs (hunk 1 above)
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:5]:
```suggestion
      Name="SwiftFormat"
```

File: installer-scripts/SwiftFormat.wxs [hunk 2]
```diff
@@ -2,42 +2,41 @@
...
       UpgradeCode="98e01ac8-a17d-43fd-99ed-1cd8b58715bf"
       Version="$(var.ProductVersion)"
       Scope="perMachine">
-    <SummaryInformation Description="SwiftFormat for Windows x86_64" />
+    <SummaryInformation Description="SwiftFormat for Windows $(var.ProductArchitecture)" />
```
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:9]:
```suggestion
    <SummaryInformation Description="SwiftFormat" />
```

File: installer-scripts/SwiftFormat.wxs [hunk 3]
```diff
@@ -2,42 +2,41 @@
...
 
     <!-- NOTE(compnerd) use pre-3.0 schema for better compatibility. -->
     <Media Id="1" Cabinet="SwiftFormat.cab" EmbedCab="yes" />
 
     <!-- Directory Structure -->
+    <!-- Components -->
+
```
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:16]: I don't think that this is useful anymore as it is inlined.

```suggestion
```

File: installer-scripts/SwiftFormat.wxs [hunk 4]
```diff
@@ -2,42 +2,41 @@
...
         <Directory Id="Library" Name="Library">
           <Directory Id="Developer" Name="Developer">
             <Directory Id="Tools" Name="Tools">
+              <Component Id="swiftformat.exe">
+                <File Id="swiftformat.exe" Source="$(var.SWIFTFORMAT_BUILD)\swiftformat.exe" Checksum="yes" />
+              </Component>
             </Directory>
           </Directory>
         </Directory>
+        <Component Id="EnvironmentVariables" Guid="b46687c3-f836-47e5-9b43-d9fd2552a731">
```
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:31]:
```suggestion

        <Component Id="EnvironmentVariables" Guid="b46687c3-f836-47e5-9b43-d9fd2552a731">
```

File: installer-scripts/SwiftFormat.wxs [hunk 5]
```diff
@@ -2,42 +2,41 @@
...
+        <!-- <Component Id="EnvironmentVariables"> -->
```
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:32]:
```suggestion
```

File: installer-scripts/SwiftFormat.wxs [hunk 6]
```diff
@@ -2,42 +2,41 @@
...
-      </Component>
-    </ComponentGroup>
-
-    <Component Id="EnvironmentVariables" Directory="INSTALLDIR" Guid="b46687c3-f836-47e5-9b43-d9fd2552a731">
-      <Environment Id="Path" Action="set" Name="Path" Part="last" Permanent="no" System="yes" Value="[INSTALLDIR]Library\Developer\Tools" />
-    </Component>
-
-    <Feature Id="SwiftFormat" AllowAbsent="no" AllowAdvertise="yes" ConfigurableDirectory="INSTALLDIR" Description="SwiftFormat for Windows x86_64" Level="1" Title="SwiftFormat (Windows x86_64)">
-      <ComponentGroupRef Id="SwiftFormat" />
+    <Feature Id="SwiftFormat" AllowAbsent="no" AllowAdvertise="yes" ConfigurableDirectory="INSTALLDIR" Description="SwiftFormat for Windows $(var.ProductArchitecture)" Level="1" Title="SwiftFormat (Windows $(var.ProductArchitecture))">
```
Comment (compnerd) [inline installer-scripts/SwiftFormat.wxs:38]:
```suggestion
    <Feature Id="SwiftFormat" AllowAbsent="no" AllowAdvertise="yes" ConfigurableDirectory="INSTALLDIR" Description="SwiftFormat" Level="1" Title="SwiftFormat">
```

**Comment by compnerd** [comment]
> Congratulations! 🎉 You just became the new WiX expert on the team! Now @compnerd is going to assume you can fix any and all WiX issues 😉.

Seems like a completely reasonable assumption to me!  It is how it works isn't it? 😆

File: installer-scripts/SwiftFormat.wxs (hunk 1 above)
Comment (tristanlabelle) [reply installer-scripts/SwiftFormat.wxs:5]: +1! Drop 'em arches!

File: installer-scripts/SwiftFormat.wxs (hunk 6 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

File: installer-scripts/SwiftFormat.wxs (hunk 5 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

File: installer-scripts/SwiftFormat.wxs (hunk 4 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs:29]: Done

File: installer-scripts/SwiftFormat.wxs (hunk 3 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

File: installer-scripts/SwiftFormat.wxs (hunk 2 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

File: installer-scripts/SwiftFormat.wxs (hunk 1 above)
Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

Comment (hjyamauchi) [reply installer-scripts/SwiftFormat.wxs]: Done

**Review by compnerd — APPROVED** [review]

**Pull request merged**